wzdx/tools/wzdx_translator.py

```diff
@@ -80,6 +81,11 @@ def parse_xml(inputfile):
         return inputfile_obj
 
 
+def parse_xml_to_dict(xml_string):
```

Comment:
Looks like we're using this method instead of `parse_xml`. Can we remove that old method and switch out its unit tests to point to this one instead?